### PR TITLE
Line 6 Relay G10S & UA Interfaces

### PR DESCRIPTION
**Pull request by Chandler-ME**
Accidentally started to edit the main data file before forking.

### CODE REVIEW

File: public/data/pedals.json [hunk 1]
```diff
@@ -4976,6 +4976,13 @@
         "Height":6.5,
         "Image":"line6-m9.png"
     },
+    {
+        "Brand":"Line6",
+        "Name":"Relay Wireless G10S"
```
Comment (tehtrav) [inline public/data/pedals.json]: This line is missing a comma at the end, and if committed will break the dropdown.

**Comment by tehtrav** [comment]
Do you have photos you're going to upload? I didn't see them in your commits

**Comment by Chandler-ME** [comment]
I'm an idiot and forgot to fork the main project before editing the data file. I then forked it and added my stuff, but it seems like that didn't exactly work. I just tried adding again, so hopefully that worked.

**Comment by tehtrav** [comment]
Haha. Git makes us all feel like idiots at times. No worries, thanks a bunch for the help! Merging this in.

**Pull request merged**